.eslintrc.json

```diff
@@ -23,6 +23,7 @@
     "consistent-return": "off",
     "no-mixed-operators": "off",
     "no-prototype-builtins": "off",
+    "no-multiple-empty-lines": ["error", { "max": 2, "maxEOF": 0 } ],
```

Comment:
@freitagbr We can also make `"max": 1`, but that involves changing:

https://github.com/shelljs/shelljs/blob/902e49c0f81834c4da61ace740cc285417b3c5d3/src/tempdir.js#L25-L29

and

https://github.com/shelljs/shelljs/blob/902e49c0f81834c4da61ace740cc285417b3c5d3/src/test.js#L17-L21

Reply:
2 lines max is fine. `"maxBOF": 0` should be added.

Reply:
Done